zio-spark-core/src/main/scala-2.12/zio/spark/SparkContext.scala

```diff
@@ -43,15 +43,7 @@ final case class SparkContext(underlying: UnderlyingSparkContext) { self =>
   def get[U](f: UnderlyingSparkContext => U): U = f(underlying)
 
   // Handmade functions specific to zio-spark
-  // template:on
-  /**
-   * Returns an immutable map of RDDs that have marked themselves as
-   * persistent via cache() call.
-   *
-   * @note
-   *   This does not necessarily mean the caching or computation was
-   *   successful.
-   */
```

Comment:
@ahoy-jon I don't know why but the generated code doesn't contain these comments.

I had to update `scalameta` to 4.9.9 otherwise there was a version clash between some sub-dependencies with `sbt-ci-release` 

That's probably the reason why we don't have these comments anymore. It's maybe an option to activate in this version of `scalameta`? I don't know. Never worked with `scalameta`

Reply:
true, will fix later